application/common/config/main.php

```diff
@@ -24,11 +25,26 @@
 
 $mfaNumBackupCodes = Env::get('MFA_NUM_BACKUPCODES', 10);
 
+$envValidator = function($name, $key, $config) {
+    if (! isset($config[$key])) {
+        $message = "The $name environment variable is required";
+        throw new EnvVarNotFoundException($message);
+    }
+};
```

Comment:
If you wanted to, you could also make a function to load and validate at the same time, something like:
```suggestion
function loadConfig($name, $requiredKeys) {
    $config = Env::getArrayFromPrefix($name);
    foreach ($requiredKeys as $key) {
        if (! isset($config[$key])) {
            $message = "The {$name}{$key} environment variable is required";
            throw new EnvVarNotFoundException($message);
        }
    }
    return $config
};
```

Reply:
Thanks. I think the way I have it is easier to follow and may even take fewer lines in the end.